docs/changelog/108896.yaml

```diff
@@ -1,6 +1,20 @@
 pr: 108896
-summary: Introduce `logs` index mode as Tech Preview
+summary: Introduce logs index mode as Tech Preview
 area: Logs
 type: feature
 issues:
  - 108896
+highlight:
+  title: Introduce logs index mode as Tech Preview
+  body: |-
+    This change introduces a new index mode named: `logs`.
+    When the new index mode is enabled then the following storage savings features are enabled automatically:
+    * Synthetic source, which omits storing the _source. When _source or part of it is requested it is synthesized on the fly at runtime.
+    * Indexing sorting. By default indices are sorting by `host.name` and `@timestamp` fields at index time. This can be overwritten if other sorting fields yield better compression rate.
```

Comment:
```suggestion
    * Index sorting. By default indices are sorted by `host.name` and `@timestamp` fields at index time. This can be overwritten if other sorting fields yield better compression rate.
```